Source/WebCore/platform/ScrollTypes.h

```diff
@@ -353,6 +354,7 @@ WEBCORE_EXPORT WTF::TextStream& operator<<(WTF::TextStream&, OverflowAnchor);
 WEBCORE_EXPORT WTF::TextStream& operator<<(WTF::TextStream&, ScrollDirection);
 WEBCORE_EXPORT WTF::TextStream& operator<<(WTF::TextStream&, ScrollGranularity);
 WEBCORE_EXPORT WTF::TextStream& operator<<(WTF::TextStream&, NativeScrollbarVisibility);
+WEBCORE_EXPORT WTF::TextStream& operator<<(WTF::TextStream&, ScrollbarWidth);
```

Comment:
This doesn't need WEBCORE_EXPORT unless it's used outside of WebCore.

Reply:
I'm not sure if it will be? I thought I'd go with matching the others. Happy to remove?

Reply:
Removed